### PR TITLE
Flips the image.

### PR DESCRIPTION
**Pull request by Trigur**

### CODE REVIEW

File: src/Manipulations.php [hunk 1]
```diff
@@ -22,6 +22,10 @@ class Manipulations
     const ORIENTATION_90 = 90;
     const ORIENTATION_180 = 180;
     const ORIENTATION_270 = 270;
+    
+    const FLIP_H = 'h';
```
Comment (freekmurze) [inline src/Manipulations.php]: rename to `FLIP_HORIZONTALLY`

File: src/Manipulations.php [hunk 2]
```diff
@@ -22,6 +22,10 @@ class Manipulations
...
+    const FLIP_V = 'v';
```
Comment (freekmurze) [inline src/Manipulations.php]: rename to `FLIP_VERTICALLY`

**Comment by Trigur** [comment]
https://github.com/spatie/docs.spatie.be/pull/287

**Comment by freekmurze** [comment]
I'll take care of the remarks myself, thanks!

**Pull request merged**